logos/src/logos/main.py

```
@@ -3364,6 +3371,129 @@ async def logosnode_reconfigure_lane(data: LogosNodeReconfigureLaneRequest):
+            continue
+
+        deadline = asyncio.get_running_loop().time() + per_model_timeout_seconds
+        while True:
+            await asyncio.sleep(poll_interval_seconds)
+            try:
+                status = await _logosnode_registry.send_command(
+                    provider_id,
+                    "get_calibration_status",
+                    timeout_seconds=10,
+                )
+            except (LogosNodeOfflineError, LogosNodeCommandError, Exception):
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)